build/art.go

```diff
@@ -88,12 +88,14 @@ func globalFlags(ctx android.LoadHookContext) ([]string, []string) {
 		cflags = append(cflags,
 			"-DART_STACK_OVERFLOW_GAP_arm=8192",
 			"-DART_STACK_OVERFLOW_GAP_arm64=16384",
+			"-DART_STACK_OVERFLOW_GAP_riscv64=16384",
 			"-DART_STACK_OVERFLOW_GAP_x86=16384",
 			"-DART_STACK_OVERFLOW_GAP_x86_64=20480")
 	} else {
 		cflags = append(cflags,
 			"-DART_STACK_OVERFLOW_GAP_arm=8192",
 			"-DART_STACK_OVERFLOW_GAP_arm64=8192",
+			"-DART_STACK_OVERFLOW_GAP_riscv64=8192",
```

Comment:
- Please sync with RVI source with riscv64-android-12.0.0_dev, because we have started the upstream to RVI source task. You can update it, make it the same as that  of RVI repo, except below point:
  
- regard the line 98, I see RVI source use 16384. I think we need to double check with RVI(T-head) before lock it down.